package.json

```diff
@@ -39,13 +39,13 @@
     "google-closure-compiler": "20161024.3.0",
     "gulp": "3.9.1",
     "gulp-clang-format": "1.0.25",
-    "gulp-tslint": "8.1.2",
-    "gulp-typescript": "3.2.4",
-    "jasmine": "2.9.0",
+    "gulp-tslint": "8.1.3",
+    "gulp-typescript": "4.0.1",
+    "jasmine": "3.0.0",
```

Comment:
Wouldn't you expect they changed API with this release?

Reply:
Sure, maybe they did, why do you ask?

Reply:
I guess I'm surprised this change works

Reply:
I would expect most semver majors to not totally break all their users, we probably only touch the mainstream stable parts of Jasmine?
Anyway, when it's green it must be good :)